05.ls/ls_1.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+file_list = Dir.glob('*')
+max_value = file_list.map(&:size).max
+files = []
+number_of_files = file_list.size
+n = if (number_of_files % 3).zero?
+      number_of_files.to_f / 3
+    else
+      number_of_files.to_f / 3 + 1.ceil
+    end
+files = file_list.each_slice(n).to_a
+
+most_common_elements = files.max_by(&:size).size
+files = files.each do |file_element|
```

Comment:
`<<`は破壊的変更を伴うメソッドなので、再代入しなくてもfilesの中身が変更されています。

```suggestion
files.each do |file_element|
```

Reply:
``` files = files.each do |file_element| ``` の再代入している部分を削除して ``` files.each do |file_element| ``` としました。